src/main/java/uk/gov/companieshouse/api/testdata/controller/TestDataController.java

```diff
@@ -71,6 +69,8 @@ public class TestDataController {
     private CompanyAuthCodeService companyAuthCodeService;
     @Autowired
     private AccountPenaltiesService accountPenaltyService;
```

Comment:
Can we put some separators between these beans please - I find it really hard to read 👎 
    @Autowired
    private TestDataService testDataService;
    @Autowired
    private CompanyAuthCodeService companyAuthCodeService;
    @Autowired
    private AccountPenaltiesService accountPenaltyService;
    @Autowired
    private VerifiedIdentityService <IdentityVerificationData>verifiedIdentityService;